content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,245 @@
+
+To create a new service product, go to :menuselection:`Sales --> Products --> Products` and click
+:guilabel:`New`. This will reveal a blank product detail form.
+
+On the new product form, add a :guilabel:`Product Name`, and set the :guilabel:`Product Type` to
+:guilabel:`Service`. Then, set the :guilabel:`Invoicing Policy` to :guilabel:`Prepaid/Fixed Price`.
+This means an invoice can be generated (and payment can be received) for this product before any
+timesheets entries have been recorded for these services.
+
+.. image:: track_and_bill/track-bill-product-prepaid-fixed.png
+   :align: center
+   :alt: View of a service product with the invoicing policy set to 'prepaid/fixed'.
+
+Finally, set the :guilabel:`Sales Price`, and confirm that the :guilabel:`Unit of Measure` is set to
+:guilabel:`Hours`.
+
+Invoice prepaid support services
+================================
+
+When support services are billed on a fixed price, an invoice can be created before any work is
+completed on the issue. In this case, a service product with the invoicing policy set to
+:guilabel:`Prepaid/Fixed Price` (like the one created above) would be used.
+
+Create a sales order with prepaid product
+-----------------------------------------
+
+To invoice a customer for prepaid support services, first create a sales order (SO) with the support
+services product. To do this, go to :menuselection:`Sales --> Orders --> Quotations --> New`, which
+reveals a blank quotation form.
+
+Then, fill out the quotation form with the customer information.
+
+Go to the :guilabel:`Order Lines` tab of the quotation and click :guilabel:`Add a Product`. Then,
+select the *prepaid services product* configured in the steps above. Update the :guilabel:`Quantity`
+field with the number of hours.
+
+After updating any other necessary information, :guilabel:`Confirm` the quotation. This converts the
+quotation into a :abbr:`SO (sales order)`.
+
+Create and send an invoice for prepaid services
+-----------------------------------------------
+
+Once the :abbr:`SO (sales order)` has been confirmed, click the :guilabel:`Create Invoice` button.
+This will open a :guilabel:`Create Invoices` pop-up window.
+
+If no down payment will be collected, the :guilabel:`Create Invoice` type can remain as
+:guilabel:`Regular Invoice`. If a down payment will be collected, choose between either
+:guilabel:`Down payment (percentage)` or :guilabel:`Down payment (fixed amount)`.
+
+When the necessary information has been entered, click :guilabel:`Create Draft Invoice`.
+
+The invoice can then be sent to the customer for payment.
+
+Create helpdesk ticket for prepaid services
+-------------------------------------------
+
+To create a *Helpdesk* ticket for prepaid services, navigate to :menuselection:`Helpdesk` and select
+a :guilabel:`Team`. This will reveal that team's :guilabel:`Pipeline`. Click :guilabel:`New` to
+create a new ticket.
```

Comment:
While I agree that pipeline does need a guilabel, in this case, I do believe 'team' deserves a guilabel, because I am referring to the kanban card for a team on the main Helpdesk dashboard. 
I understand it may seem inconsistent that sometimes team is labeled and sometimes it is not, but is usually the case of referring to a team in general or directing the user to select a team as part of an action.